auth-server/src/services/auth.ts

```diff
@@ -448,6 +461,10 @@ export default class AuthService {
 			currentRole = Role.PROPOSAL_BOT;
 		}
 
+		const kusamaAddresses = addresses
```

Comment:
This needs to be of the form `{"address1","address2"}`

Reply:
done